next.config.js

```diff
@@ -18,14 +17,10 @@ module.exports = withPlugins([[indexSearch], [feed], [sitemap], [socialImages]],
   // verbose: true,
 
   env: {
-    WORDPRESS_GRAPHQL_ENDPOINT: removeLastTrailingSlash(process.env.WORDPRESS_GRAPHQL_ENDPOINT),
+    WORDPRESS_GRAPHQL_ENDPOINT: process.env.WORDPRESS_GRAPHQL_ENDPOINT,
```

Comment:
I just realized you could skip this step, removing this line and use at each plugin the env variable directly (as it doesnt have any modification or default): 
```
new WebpackPluginCompiler({          
     url: process.env.WORDPRESS_GRAPHQL_ENDPOINT,
     ....
```

Reply:
thats a really great point haha i wonder if you think though there's a possibility we would want to create a plugin using a different graphql source? would that even work?

though maybe that's not worth supporting "just in case" as an edge case and for the starter sake the simplicity is better 🤔

Reply:
Good question, didnt think of that... Doing it the way is now, the plugin will maintain its decoupled and independant condition (as it works with the explicit arguments) which I think is always a good idea. Probably it only depends on the plugins desired scope, and if they will work in other contexts (not relying exclusively on env variables)

Reply:
ill just leave it for now then, we can discuss it more later